docker/package/wizard_structure.py

```diff
@@ -416,10 +424,14 @@ def pprint_options(self):
                     i += 1
         elif self.options and isinstance(self.options, dict):
             index_len = len(str(len(self.options)))
-            str_format = f"{{:{index_len}}}. {{:<26}}  {{}}"
+            max_option_len = max(map(len, self.options.keys()))
+            padding = max(26, max_option_len + 2)
+            indent_size = index_len + 2 + padding
+            str_format = f"{{:{index_len}}}. {{:<{padding}}}  {{}}"
             for o in self.options:
                 description = textwrap.indent(
-                    textwrap.fill(self.options[o], 60), " " * 31
+                    textwrap.fill(self.options[o], int(padding * (60 / 26))),
```

Comment:
Oh nice, that's clever :+1: Unfortunately, I don't think it'll work well. This makes the right half of the options listing (all the descriptions) approximately twice (`60/26` times) as long as the left part. Works out to 60 characters in most cases, but precisely when the options are very long, like with URLs, this will stretch the description out a lot and make it ill-indented or not fit onto most screens. See what I mean:

```
>>> max_option_len = 80
>>> padding = max(26, max_option_len + 2)
>>> indent_size = 1 + 2 + padding
>>> print(textwrap.indent(textwrap.fill("a" * 400, 60), " " * indent_size))
                                                                                     aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
                                                                                     aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
                                                                                     aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
                                                                                     aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
                                                                                     aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
                                                                                     aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
                                                                                     aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

>>> print(textwrap.indent(textwrap.fill("a" * 400, int(padding * (60/26))), " " * indent_size))
                                                                                     aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
                                                                                     aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
                                                                                     aaaaaaaaaaaaaaaaaaaaaa
```

Leaving the width at 60 is fine IMO, it's the indentation of it that matters for the padding anyway, which you have already fixed.

Reply:
Actually the idea was to make it shorter, but I somehow wrote the oblivosly wrong formula 🙈

Reply:
Hmm, still doesn't look well in some cases
```
Do you want to set up baking or to run the standalone node?
1. baking (default)            Set up and start all services for baking: tezos-node and
                             tezos-baker.
2. node                        Only bootstrap and run the Tezos node.
```

Reply:
Looks like `index_len` indent is missing

Reply:
Indent for dot + space was missing actually